### DISCUSSION TITLE
Doc comment on Search operators (/api/graphql/search-operators)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1903] [Docs/blog comments]
# Doc comment on Search operators (/api/graphql/search-operators)

Overview

https://weaviate.io/developers/weaviate/api/graphql/search-operators

**Comment by edwineve** [comment]
What is the default moveTo/moveFrom force?

**Reply by databyjp** [reply]
Hi there, I believe if you use `moveTo or `moveFrom`, a force is required.
